fiduceo_uncertainties.f90

```diff
@@ -1539,24 +1537,24 @@ SUBROUTINE radiance_uncertainties(i,outData,coefs1,coefs2,coefs3,FCDR,&
                FCDR%dre_over_drict3(j,i)**2*FCDR%urict3_r(i)**2)
        end if
 
-       if ((FCDR%ucict3(i) .ne. NAN_R) &
-            .and. (FCDR%ucict3(i) .gt. 0) &
-            .and. (FCDR%ucs3(i) .ne. NAN_R) &
-            .and. (FCDR%ucs3(i) .gt. 0) &
+       if ((FCDR%ucict3 .ne. NAN_R) &
```

Comment:
And back for the correct uncertainty handling

Reply:
This part looks ok

--> no change